openvdb_houdini/GEO_PrimVDB.h

```diff
@@ -429,6 +429,11 @@ class OPENVDB_HOUDINI_API GEO_PrimVDB : public GEO_Primitive
     void                        makeGridUnique()
                                     { myGridAccessor.makeGridUnique(); }
 
+    /// @brief Returns if the tree is not shared.  If it is not shared,
```

Comment:
Returns _**true**_ if the tree...

Thanks for adding this. I think I might even add this check to GridBase as it seems pretty useful.

Reply:
Note that at the GridBase level it should be called isTreeUnique.